fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneRecordCursor.java

```diff
@@ -124,6 +124,10 @@
         }
         this.fields = fields;
         this.groupingKey = groupingKey;
+        KeyExpression sort = ((LuceneExecuteProperties)scanProperties.getExecuteProperties()).sort;
+        ExecutorService service = ((LuceneExecuteProperties) scanProperties.getExecuteProperties()).service;
+        if (sort != null) setSort(sort.toKeyExpression());
+        if (service != null) setExecutor(service);
```

Comment:
I guess with this change, the public methods ```setSort``` and ```setExecutor``` are not needed to be exposed any more?